tests/test_project/tests/test_qa.py

```diff
@@ -118,6 +118,13 @@ def test_qa_call_check_commit_message_pass(self):
                 '[qa] Improved Y #20\n\n'
                 'Simulation of a special unplanned case\n\n#noqa',
             ],
+            [
+                'commitcheck',
+                '--quiet',
+                '--message',
+                '[fix] Fixed extensibility of openwisp-users and added sample_users test app #377\n\n'
+                'Closes #377\r\n\r\nCo-authored-by: Ajay Tripathi <ajay39in@gmail.com>',
```

Comment:
This is the commit message I copied from openwisp-controller for which tests are failing